### ISSUE TITLE
Backport NAV fixes

### ISSUE THREAD

**Issue by hbeni** [c172p-team/c172p#1485]
Hi, is it possible to backport the current nav gauges back to the stable version?

https://github.com/c172p-team/c172p/pull/1360

It is currently not so easy to do radio navigation, because:
A) the needle rotation center is wrong
B) there are no yellow ticks, so selecting an OBS is relying on the feedback of the tooltip

**Comment by wlbragg** [comment]
Yeah, I can cherry pick this to our 2020.3.19 and ask Stuart to backport it to fgdata 2020.3.19.
